### DISCUSSION TITLE
[Question] Comparison of GPU utilization between Isaac lab and Isaac gym

### DISCUSSION THREAD

**Discussion by xliu0105** [isaac-sim/IsaacLab#743] [Q&A]
### Question

When I run a quadruped robot training with 4096 environments on isaac gym, the GPU utilization is about 60%, while when I use isaac lab to train the same robot with the same environment, the GPU utilization is only about 30%. Is this because isaac lab has better computational optimization or because it cannot fully utilize the performance of the GPU?

**Comment by Mayankm96** [comment]
If you are comparing the rough terrain locomotion training, it is likely the memory consumption is reduced due to the usage of the `trimesh` library for some of the terrain generation (instead of height-field generation). The generated terrain in Isaac Lab may have a lesser number of triangle faces, so it consumes less memory accordingly.

However, some proper benchmarking is needed to make a more general statement.
